### PR TITLE
small edits

### PR DESCRIPTION
**Pull request by brtnfld**

### CODE REVIEW

**Comment by brtnfld** [comment]
I copy src/H5Tconv.c from develop, still shows a difference.
I run bin/format_source, H5Tconv.c still shows a difference.

**Review by derobins — APPROVED** [review]

**Review by qkoziol — APPROVED** [review]

**Comment by brtnfld** [comment]
I think my version of src/H5Tconv.c is correctly formatted.

**Comment by lrknox** [comment]
> 
> 
> I think my version of src/H5Tconv.c is correctly formatted.

Yes, it passed the format check.  The only thing blocking merge is a review from one of the code owners.  I'll take a look - if I can make sense of the changes I'll approve and merge it.

**Review by lrknox — APPROVED** [review]

**Pull request merged**
